src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -1152,6 +1152,14 @@ private bool isValidStringRef(CORINFO_MODULE_STRUCT_* module, uint metaTOK)
         private bool shouldEnforceCallvirtRestriction(CORINFO_MODULE_STRUCT_* scope)
         { throw new NotImplementedException("shouldEnforceCallvirtRestriction"); }
 
+        private short* getStringLiteral(CORINFO_MODULE_STRUCT_* module, uint metaTOK, ref int length)
```

Comment:
Nit: It would be natural for this to return `char*`. (Change the mapping in ThunkInput.txt.)

Reply:
@jkotas should I change short to char for all of these: https://github.com/dotnet/runtime/blob/master/src/coreclr/src/tools/Common/JitInterface/ThunkGenerator/ThunkInput.txt#L75-L79 ?

Reply:
Done in https://github.com/dotnet/runtime/pull/1378/commits/0940e93ecd4af0fcd7731c9feefaa91cc9ef7c7b

Reply:
Thanks!